processor/src/stack/tests.rs

```diff
@@ -39,21 +37,39 @@ fn initialize() {
 
 #[test]
 fn initialize_overflow() {
```

Comment:
nit: maybe we should rename this test since this is not about initializing any more.